pom.xml

```diff
@@ -77,7 +77,7 @@
         <google-guava.version>19.0</google-guava.version>
         <log4j2.version>2.17.1</log4j2.version>
         <jackson.core.version>2.9.10</jackson.core.version>
-        <tomcat-api7.version>7.0.8</tomcat-api7.version>
+        <tomcat-api7.version>7.0.109</tomcat-api7.version>
```

Comment:
Is there any information on what this upgrade provides. Looks to be a pretty significant version boost.

Reply:
I did some quick rudimentary performance testing and saw about some performance for basic boolean logic rules and actually saw improvements in performance for ones which contain functions. I think this MR is good to merge. @stevenchen-indeed if you agree I can merge and deploy this on Monday 4/17